sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_shared/models.py

```diff
@@ -12,7 +12,8 @@ class DeprecatedEnumMeta(CaseInsensitiveEnumMeta):
 
     def __getattribute__(cls, item):
         if item == "MICROSOFT_BOT":
-            warnings.warn("MICROSOFT_BOT is deprecated and should not be used.", DeprecationWarning)
+            warnings.warn("MICROSOFT_BOT is deprecated and has been replaced by \
+                          MICROSOFT_TEAMS_APP identifier.", DeprecationWarning)
```

Comment:
```suggestion
                          MICROSOFT_TEAMS_APP identifier.", DeprecationWarning)
              return CaseInsensitiveEnumMeta.__getattribute__(cls, 'MICROSOFT_TEAMS_APP')
```

as MICROSOFT_BOT will be removed, we have to return MICROSOFT_TEAMS_APP instead

Reply:
Changed.